csgc-jekyll-src/meetings.md

```diff
@@ -8,7 +8,7 @@ permalink: /meetings/
 * *When*: Generally, alternating Mondays, 5:30pm-6:30pm. Meetings for this semester are 9/9, 9/23, 10/7, 10/21, 11/4, 11/18, and 12/2
 * *Where*: Torgersen 3160A (If the door is locked, please knock and someone will let you in).
 
-See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info.
+See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info (**please note some faculty are on the mailing list**).
```

Comment:
```suggestion
See the [mailing list](https://groups.google.com/a/vt.edu/forum/#!forum/csgc-g) for the latest info (**please note this is an open list, and may include faculty**).
```

Reply:
I inserted a suggestion using Jamie's wording, since the issue is less that _faculty_ are on the list, and more that it's open to _anyone_.

To answer your question: yes, those are the correct emails, and the new list has been tested.